src/index.html

```diff
@@ -9,10 +9,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
You should link CSS files to HTML, not SCSS. Browsers don't know how to interpret SCSS files. You need to compile your SCSS files into CSS and link the generated CSS file here.